tests/components/xiaomi_miio/test_select.py

```diff
@@ -0,0 +1,410 @@
+"""The tests for the xiaomi_miio select component."""
+
+import logging
+
+from arrow import utcnow
+from miio.airfresh_t2017 import (
+    DisplayOrientation as AirfreshT2017DisplayOrientation,
+    PtcLevel as AirfreshT2017PtcLevel,
+)
+from miio.airhumidifier import LedBrightness as AirhumidifierLedBrightness
+from miio.airhumidifier_miot import LedBrightness as AirhumidifierMiotLedBrightness
+from miio.tests.test_airfresh_t2017 import DummyAirFreshT2017
+from miio.tests.test_airhumidifier import DummyAirHumidifier
+from miio.tests.test_airhumidifier_miot import DummyAirHumidifierMiot
```

Comment:
Let's avoid importing anything from python-miio here, and simply create some dummy enums that fulfill the same purpose.

My personal opinion is that the tests should verify that `XiaomiGenericSelector` and relevant pieces work in `select.py`, no matter which device is exposing those.

But I'll take another look at the test code soon, I just wanted to give you some quick comments asap.

Reply:
@rytilahti I dont really understand how to do this with out import Dummy device. (Maybe only ones?) Because I need coordinator to check _handle_coordinator_update function. And other milo interfaces.

Maybe if I remove airhumidifier_miot , test_airfresh_t2017 and use only airhumidifier class it will be enough?

Reply:
@rytilahti Tests was updated. Could you look at please.